pydruid/utils/postaggregator.py

```diff
@@ -98,3 +98,56 @@ def __init__(self, name):
         Postaggregator.__init__(self, None, None, name)
         self.post_aggregator = {
             'type': 'hyperUniqueCardinality', 'fieldName': name}
+
+
+
+
+class ThetaSketchOp:
+    def __init__(self, fn, fields, name):
+        self.post_aggregator = {'type': 'thetaSketchSetOp',
+                                'name': name,
+                                'func': fn,
+                                'fields': fields}
+        self.name = name
+
+    def __or__(self, other):
+        return ThetaSketchOp('UNION', self.fields(other),
+                              self.name + 'or' + other.name)
```

Comment:
minor nit: maybe change name to self.name + '_or_' + other.name, in order to make names more readable. 
same comment for and and not